extensions/common/aws/aws-test/src/testFixtures/java/org/eclipse/dataspaceconnector/aws/testfixtures/AbstractS3Test.java

```diff
@@ -83,15 +85,25 @@ static void prepareAll() {
                 .pollInterval(Duration.ofSeconds(2))
                 .ignoreException(IOException.class) // thrown by pingMinio
                 .ignoreException(ConnectException.class)
-                .until(AbstractS3Test::pingMinio);
+                .until(() -> {
```

Comment:
minor nit: introducing a method, something like `isBackendAvailable` or similar naming would make this piece of code more readable:
```
.ignoreException(ConnectException.class)
.until(this::isBackendAvailable)

// and then:
private boolean isBackendAvailable(){
   if(isMinio()){
      return isMinioAvailable();
   } else {
      return true;
   }
};
```

Reply:
Sure. I updated the code.